### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by leonardodiasb**
Created Desktop version in a media query from the mobile first version css.
Added a list with links in the navbar.
Changed content by duplicating the html on the work section, adding a display "none" for the "works-section-desktop" on mobile CSS part, and display "none" for the "works-section" in the media query.
Applied hover function to each container on the works section.
Applied CSS on the other sections.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -406,3 +426,283 @@ textarea[type="text"] {
   margin-bottom: 45px;
   margin-left: 24px;
 }
+
+#works-section-desktop {
+  display: none;
+}
+
+@media (min-width: 500px) {
```
Comment (vichuge) [inline style.css]: - [ ] Please, use media queries for the Desktop breakpoint in the CSS file.(min-width > 992px - 1100px), for example `@media screen and (max-width: 1080px) {` 🐶

File: index.html [hunk 2]
```diff
@@ -17,12 +17,17 @@
...
             <a href="#" id="logo">My Logo</a>
             <a href="#" class="menu-icon"><img src="images/Union.png" alt="menu bar"></a>
+            <ul class="menu">
+                <li class="menu-items"><a href="#">Portfolio</a></li>
+                <li class="menu-items"><a href="#">About</a></li>
+                <li class="menu-items"><a href="#">Contact</a></li>
+            </ul>
         </nav>
     </header>
     <section id="headline">
```
Comment (vichuge) [inline index.html:27]: - [ ] The background image on this section appears at the middle of the screen on desktop size, please check this little but important issue.
image

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @leonardodiasb, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

mario_ladders

## Good points 👍 
- No Linters errors ✔️ 
- Good job for design ✔️ 

## Aspects to improve ♻️

- [ ] It looks like the form has the mobile size on desktop, please fix this
image

- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, consider:
- Use more details according to Figma style, image below:
image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by leonardodiasb** [comment]
Hello Victor ! On this first image of the Contact Me section I am pretty sure I corrected it on this Desktop-Version branch as you can see here:
reply1
reply2

And I also added those background images:
reply3
raply4

I'm sorry for asking, but did I send the url from the last commit I did today on Microverse Dashboard ?

**Review by Shaher-11 — APPROVED** [review]
Hello @leonardodiasb ,

# Congratulations! 🎉 
#### Your project is approved!  There is nothing else to say other than... it's time to merge it :muscle: :muscle: 

## Optional Suggestions 💡
_The following point Is optional to implement however it's highly recommended!_:blush:

- Add some space between the about me paragraph and let's conncet

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

**Pull request merged**